tcpdevmem.c

```diff
@@ -11,6 +11,44 @@
 
 #define TEST_PREFIX "ncdevmem"
 
+int driver_reset(const struct options *opts) {
+	char driver_reset_cmd[512];
+	int ret = 0;
+
+	sprintf(driver_reset_cmd, "ethtool --set-priv-flags %s enable-strict-header-split on", opts->tcpd_link_name);
+	ret = ret | system(driver_reset_cmd);
```

Comment:
Please change to:

if (ret)
   return ret;

to early exit on error, instead of continuing on error and returning an error later.